File: src/test/java/common/Candidatos.java
package common;

import model.Candidato;

public enum Candidatos {

	Rose(1L, "Lisa Rose", new Candidato()),
	Seymour(2L, "Gene Seymour", new Candidato()),
	Phillips(3L, "Michael Phillips", new Candidato()),
	Puig(4L, "Claudia Puig", new Candidato()),
	LaSalle(5L, "Mick LaSalle", new Candidato()),
	Matthews(6L, "Jack Matthews", new Candidato()),
	Toby(7L, "Toby", new Candidato());

	private Long id;
	private final String name;
	private final Candidato candidato;

	Candidatos(Long id, String name, Candidato candidato) {
		this.id = id;
		this.name = name;
		this.candidato = new Candidato(this.id, this.name);
	}

	public String getName() {
		return name;
	}

	public Candidato getCandidato() {
		return candidato;
	}
}

File: src/test/java/common/Candidatos.kt
package common

import model.Candidato

enum class Candidatos constructor(private val id: Long?, val nome: String, candidato: Candidato) {

    Rose(1L, "Lisa Rose", Candidato()),
    Seymour(2L, "Gene Seymour", Candidato()),
    Phillips(3L, "Michael Phillips", Candidato()),
    Puig(4L, "Claudia Puig", Candidato()),
    LaSalle(5L, "Mick LaSalle", Candidato()),
    Matthews(6L, "Jack Matthews", Candidato()),
    Toby(7L, "Toby", Candidato());

    val candidato: Candidato = Candidato(this.id, this.nome)

}
